src/components/MenuButton/MenuButton.js

```diff
@@ -0,0 +1,26 @@
+import { html } from '../../shared';
+import styles from './MenuButton.module.css';
+
+/**
+ * @param {{ text: string, onClick: (e: MouseEvent) => void, variant?: 'normal' | 'outlined', disabled?: boolean }} props
+ * @returns {HTMLButtonElement}
+ */
+function MenuButton({ text, onClick, variant = 'normal', disabled = false }) {
+  const classNamesForVariant = {
+    normal: styles.menuButtonNormal,
+    outlined: styles.menuButtonOutlined,
+  };
+
+  const menuButton = html`<button
+    class="${classNamesForVariant[variant]}"
+    type="button"
+    ${disabled ? ' disabled aria-disabled="true"' : ''}
```

Comment:
Taki zapis można skrócić wykorzystując operator logiczny &&. Często w językach programowania wykorzystuje się fakt, że jeżeli pierwsza część równania operatora && jest fałszywa to nie ma sensu sprawdzać prawej strony równania, bo niezależnie od wartości logicznej, operacja zwróci fałsz. Ten fakt można wykorzystać właśnie tutaj stosując zapis `disabled && 'disabled aria-disabled=true'`. Prawa część jest zawsze prawdą, string z disabled, więc wynik zależy tylko od propa disabled. Mam nadzieje, że wszystko jasne 😅

Reply:
Na pewno? Jesteśmy tu w zwykłym stringu, więc dla `disabled = false` obecna metoda zwróci pusty string, a Twoja sugestia string `"false"`. Sprawdziłem jeszcze na szybko zmianę i generuje się taki element:
```HTML
<button class="..." type="button" false="">
    Test
</button>
```
A nie:
```HTML
<button class="..." type="button">
    Test
</button>
```

Reply:
Jasne, zostawiam jak było, dzięki za uwagi 👍